pyproject.toml

```diff
@@ -98,19 +99,7 @@ select = [
 ]
 
 ignore = [
-    "E501",    # Line too long ({width} > {limit} characters)
-    "E701",    # Multiple statements on one line (colon)
-    "E731",    # Do not assign a lambda expression, use a def
-    "E402",    # Module level import not at top of file
-    "PLR0911", # Too many return statements
-    "PLR0912", # Too many branches
-    "PLR0913", # Too many arguments in function definition
-    "PLR0915", # Too many statements
-    "PLR2004", # Magic value used instead of constant
-    "PLW0603", # Using the global statement
-    "PLW2901", # redefined-loop-name
-    "PLR1714", # consider-using-in
-    "PLR5501", # else-if-used
+    "F403", # 'from module import *' used; unable to detect undefined names
```

Comment:
```suggestion
    "E501",    # Line too long ({width} > {limit} characters)
    "E701",    # Multiple statements on one line (colon)
    "E731",    # Do not assign a lambda expression, use a def
    "E402",    # Module level import not at top of file
    "PLR0911", # Too many return statements
    "PLR0912", # Too many branches
    "PLR0913", # Too many arguments in function definition
    "PLR0915", # Too many statements
    "PLR2004", # Magic value used instead of constant
    "PLW0603", # Using the global statement
    "PLW2901", # redefined-loop-name
    "PLR1714", # consider-using-in
    "PLR5501", # else-if-used
    "F403", # 'from module import *' used; unable to detect undefined names
```